Tests/Functional/xctest_checker/xctest_checker/compare.py

```diff
@@ -28,8 +28,10 @@ def _expected_lines_and_line_numbers(path, check_prefix):
     with open(path) as f:
         for line_number, line in enumerate(f):
             if line.startswith(check_prefix):
-                yield line[len(check_prefix):], line_number+1
+                yield line[len(check_prefix):].strip(), line_number+1
 
+def _add_whitespace_leniency(original_regex):
+    return "^ *" + original_regex + " *$"
```

Comment:
Yeah, that's a good point. `\s` is probably more appropriate here considering the way that I've been referencing `whitespace` generally in the docs for this PR.

OTOH, considering that it's not actually necessary at this moment, and CI has gone green already with the existing code (on Linux at least), how about we make that adjustment separately?